dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -8307,6 +8309,15 @@ int getSelectPlan(gp_walk_info& gwi, SELECT_LEX& select_lex, SCSEP& csep, bool i
           if ((ord_item->type() == Item::CONST_ITEM && ord_item->cmp_type() == INT_RESULT) &&
               ord_item->full_name() && !strcmp(ord_item->full_name(), "Not_used"))
           {
+            // This is the case of server's optimization of EXISTS(SELECT ...).
+            // look at sql_base.cc.
+            continue;
+          }
+          else if (ord_item->type() == Item::NULL_ITEM)
+          {
+            // Do not add NULL contant as a order key.
+            // NULL as order key is an optimization hint for Mysql, but
```

Comment:
MariaDB.

Reply:
It was introduced at MySQL and for MySQL. So I attributed it to MySQL.

Reply:
This project is not MySQL anymore and moreover a real effort has been done on renaming binaries so plz change it to MariaDB to follow the common strategy.